fastlane/lib/fastlane/actions/badge.rb

```diff
@@ -16,7 +16,13 @@ def self.run(params)
           shield_gravity: params[:shield_gravity],
           shield_no_resize: params[:shield_no_resize]
         }
-        Badge::Runner.new.run(params[:path], options)
```

Comment:
Seems like this is a duplicate change from https://github.com/fastlane/fastlane/pull/9295/files?